packages/reporter/src/sessions/sessions-model.ts

```diff
@@ -15,13 +15,14 @@ export interface SessionProps extends InstrumentProps {
 }
 
 export default class Session extends Instrument {
-  @observable name: string
```

Comment:
Why did this change?

Reply:
its declared in the `Instrument` class so its inherited. declaring it twice in mobx 6 throws an error